src/Compute/Compute/Generated/VirtualMachineScaleSet/Config/SetAzureRmVmssStorageProfileCommand.cs

```diff
@@ -87,6 +87,11 @@ public partial class SetAzureRmVmssStorageProfileCommand : Microsoft.Azure.Comma
             ValueFromPipelineByPropertyName = true)]
         public string OsDiskCreateOption { get; set; }
 
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true)]
```

Comment:
I want to push for good practice at this early stage, to add "HelpMessage:" here. 
Look for other examples and the message itself, I usually get from swagger

Reply:
Got it, will do thanks